packages/laconia-config/src/index.js

```diff
@@ -2,10 +2,12 @@ const EnvVarConfigFactory = require("./EnvVarConfigFactory");
 const BooleanConfigConverter = require("./BooleanConfigConverter");
 const SsmConfigConverter = require("./SsmConfigConverter");
 const S3ConfigConverter = require("./S3ConfigConverter");
+const SecretsManagerConfigConverter = require("./SecretsManagerConfigConverter");
 
-exports.envVarInstances = () => ({ env, $ssm, $s3 }) =>
+exports.envVarInstances = () => ({ env, $ssm, $s3, $secretsManager }) =>
```

Comment:
`packages/laconia-core/src/laconia.js` would need to be updated to instantiate and register `$secretsManager`

Reply:
Is this what you mean, or am I missing a spot? https://github.com/geoffdutton/laconia/blob/issue-105-secrets-access/packages/laconia-core/src/laconia.js#L18

Reply:
My bad! I totally missed this :)